content/developer/tutorials/server_framework_101/04_relational_fields.rst

```diff
@@ -646,6 +646,11 @@ to a list of offers received from potential buyers.
    #. Allow connecting properties to multiple offers.
    #. Modify the form view of properties to display offers in a new notebook page titled "Offers".
 
+   .. tip::
+      The `default` field argument expects a callable function, not a precalculated value. If you
+      mistakenly pass the result of calling the `fields.Date.today` helper function, the field's
```

Comment:
Here I'm mentioning the call to the function and referencing it by its name, so I think I should keep it as-is.

It's different for the decorators in chapter 5 because I want to imply their invocation and reflect how they are used in the code.